public/css/pwca-print-method-modal.css

```
@@ -0,0 +1,296 @@
+  border-radius: 4px;
+  border: none;
+  cursor: pointer;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+  font-size: 14px;
+  transition: all 0.2s ease-in-out;
+  border: 1px solid #e2e8f0;
+  font-weight: 500;
+  background: #ffffff;
+  color: #64748b;
+}
+.pwca-print-method-modal__btn:focus, .modal#pwca-print-method-modal .modal__overlay .modal__container .modal__footer .modal__btn:focus {
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)